ci/caddy/Caddyfile

```diff
@@ -0,0 +1,58 @@
+# initial block determines global options
+{
+	email {$TLS_EMAIL}
+}
+
+# see https://caddyserver.com/docs/caddyfile/patterns
+*.{$ROOT_DOMAIN}, {$ROOT_DOMAIN} {
+	# wilcard certs require a DNS challenge
+	tls {
+		dns vultr {$VULTR_API_KEY}
+		propagation_delay 30s # default 0s (NB. Vultr gives 300s as DNS record TTL)
+		propagation_timeout 5m # default 2mins (-1 indicates to not conduct propagation check, just handover to ACME after delay)
+		resolvers 1.1.1.1 8.8.8.8
+	}
+
+	@root host {$ROOT_DOMAIN} www.{$ROOT_DOMAIN}
+	handle @root {
+		reverse_proxy editor:8043
+	}
+
+	@api host api.{$ROOT_DOMAIN}
+	handle @api {
+		reverse_proxy api:{$API_PORT}
+	}
+
+	@hasura host hasura.{$ROOT_DOMAIN}
+	handle @hasura {
+		reverse_proxy hasura-proxy:{$HASURA_PROXY_PORT}
+	}
+
+	@localplanning host localplanning.{$ROOT_DOMAIN}
+	handle @localplanning {
+		reverse_proxy localplanning:8043
+	}
+
+	@postgres host postgres.{$ROOT_DOMAIN}
+	handle @postgres {
+		reverse_proxy postgres:{$PG_PORT}
+	}
+
+	@sharedb host sharedb.{$ROOT_DOMAIN}
+	handle @sharedb {
+		reverse_proxy sharedb:{$SHAREDB_PORT}
+	}
```

Comment:
https://sharedb.5030.planx.pizza/

<img width="398" height="174" alt="Image" src="https://github.com/user-attachments/assets/fe26d758-0b04-4fb4-8938-37e2970eea6d" />

I can't spot an error here, but this is currently not accessible directly or via the Editor. Docker logs suggest the service is up and running.

Reply:
I think this is something to do with ports in the Caddyfile - investigating